Moq.AutoMock/Resolvers/MockResolutionContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Moq.AutoMock.Resolvers
+{
+    public class MockResolutionContext
+    {
+        public AutoMocker AutoMocker { get; set; }
```

Comment:
This should be a readonly property. I would also suspect that we should make RequestType readonly as well.